### PR TITLE
Enabling probes in the US, China, and Iran

### PR DESCRIPTION
**Pull request by myleshorton**

### CODE REVIEW

File: embeddedconfig/global.yaml.tmpl [hunk 1]
```diff
@@ -138,10 +138,16 @@ featuresenabled:
     - label: cn-shortcut
       geocountries: cn
       versionconstraints: "!7.4.0"
-  #  - label: ir-desktop-shortcut
-  #    geocountries: ir
-  #    platforms: windows,darwin,linux
+  # - label: ir-desktop-shortcut
+  #   geocountries: ir
+  #   platforms: windows,darwin,linux
   # probeproxies is not enabled anywhere
```
Comment (hwh33) [inline embeddedconfig/global.yaml.tmpl]: This comment is now out-of-date

File: embeddedconfig/global.yaml.tmpl [hunk 2]
```diff
@@ -138,10 +138,16 @@ featuresenabled:
...
+  probeproxies:
+    - label: probeir
+      geocountries: ir
+    - label: probeuscn
+      geocountries: us,cn
```
Comment (hwh33) [inline embeddedconfig/global.yaml.tmpl]: Why are these combined, but the Iran probes are separate?  I don't know much about our proxy probing logic

Comment (myleshorton) [reply embeddedconfig/global.yaml.tmpl]: I think this is just about having separate labels AFIACT, but we don't actually use the labels anywhere on the client side, so I just went ahead and consolidated them.

File: embeddedconfig/global.yaml.tmpl (hunk 1 above)
Comment (myleshorton) [reply embeddedconfig/global.yaml.tmpl]: Fixed!

**Comment by myleshorton** [comment]
Thanks @hwh33! I think I addressed those comments.

**Review by hwh33 — APPROVED** [review]
LGTM

**Pull request merged**